cmake/jsoncpp.cmake

```diff
@@ -7,7 +7,7 @@ else()
 endif()
 
 set(prefix "${CMAKE_BINARY_DIR}/deps")
-set(JSONCPP_LIBRARY "${prefix}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}jsoncpp${CMAKE_STATIC_LIBRARY_SUFFIX}")
+set(JSONCPP_LIBRARY "${prefix}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}jsoncpp_static${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
With this change we'll only support specifically 1.9.4 because the `_static` suffix is going away again in 1.9.5 (see https://github.com/ethereum/solidity/pull/11045#issuecomment-790781864).

I'd rather go with the solution from #11045. We decided not to merge it at the time because we thought we'd just skip 1.9.4 but I think it was a better solution than changing the name here.

EDIT; Just realized that we only support one particular version of json cpp rather than a range. Still, using `STATIC_SUFFIX` would make later upgrade to 1.9.5 easier.